code/datums/gamemodes/infestation.dm

```diff
@@ -1 +1,9 @@
 /datum/game_mode/infestation
+
+/datum/game_mode/infestation/scale_roles()
+	
+	. = ..()
+	if(!.)
+		return
+	var/datum/job/scaled_job = SSjob.GetJobType(/datum/job/terragov/silicon/synthetic)
+	scaled_job.job_points_needed  = 40 //For every 40 marines there will be 1 extra Synth
```

Comment:
No. This code doesn't do anything. 

You're setting that synth requires 40 job points, but you are never adding any job point to that job. 

I already told you how to do it, you either use job worth or the proc i gave you.

Reply:
Well tivi suggested this code, and I am still unsure about your jobworth proc so I am kinda figuring it out. I am gonna make it a draft till then anyway.